doc/userguide/rules/ja4-keywords.rst

```diff
@@ -0,0 +1,26 @@
+JA4 Keywords
+============
+
+Suricata comes with a JA4 integration (https://github.com/FoxIO-LLC/ja4). JA4,
+as part of the larger JA4+ suite of fingerprints, is used to fingerprint TLS
+clients.
+
+JA4 support must be enabled in the Suricata config file (set
+``app-layer.protocols.tls.ja4-fingerprints`` to ``yes``). If it is not
+explicitly disabled (``no``) , it will enabled if a loaded rule requires it.
+
+ja4.hash
+--------
+
+Match on JA4 hash (e.g. ``q13d0310h3_55b375c5d22e_cd85d2d88918``).
```

Comment:
We should mention the fact that it is a compile-time option that can be tested with `requires` keyword, cc @jasonish @jufajardini

Reply:
Added a quick note mentioning the compile time option and the use of `requires`.